paper.md

```diff
@@ -433,10 +442,20 @@ We present a few case studies using an ML workflow to demonstrate how Popper hel
 Next, we compare Popper with existing state-of-the-art workflow engines illustrating its YAML based workflow syntax that has a relatively low entry barrier and its ability to run containerized workflows without requiring access to any cloud environment.
 
 As future work, we have planned to add support for more container engines like NVIDIA Pyxis, Charliecloud, Shifter and resource managers like HTCondor, TORQUE to Popper in order to extend the range of the different computing environments currently supported.
-We also plan to add a compatibility layer between the Popper syntax and more advanced workflow languages such as CWL/WDL to enable interoperability between different workflow engines.
-This would also allow Popper users to migrate to other workflow engines without making any changes to their previously written workflows, thus keeping the overhead minimal.
-Several other features have also been planned like workflow report generation, GUI dashboard, etc. to make the tool more user friendly.
+We plan to add more exporter plugins for exporting Popper workflows to advanced workflow syntaxes such as CWL, WDL, and Airflow to enable interoperability between different workflow engines.
+We are also planning to add an `image` attribute to our workflow syntax, that would abstract the building and pushing of images irrespective of the container engine used.
+Some additional improvements planned for Popper are as follows:
 
-# References {#sec:references}
+* Currently, Popper supports logging to the STDOUT or a file. This can be extended to have an abstract mechanism to store and export logs to logging drivers like syslog, fluentd, AWS CloudWatch, etc.
+Similar abstractions can be implemented to support different tracing mechanisms of containers like `dtrace`.
 
+* Adding plugins for engines like Kata container and Firecracker will allow running containers with the isolation guarantees of a VM.
+* Caching and Layering of images is taken care of by the underlying container engine by default. We plan to add an `image` attribute to our workflow syntax to abstract the building and pushing of images using tools like Kaniko and BuildKit that supports rootless image builds and makes builds efficient using their remote caching feature.
+
+**Acknowledgements**: This work was partially funded by the NSF Award #OAC-1836650 (IRIS-HEP [^iris-hep]) and #CNS-1705021 and the Center for Research in Open Source Software (CROSS)[^cross].
```

Comment:
"funded by NSF awards #OAC-1836650 (IRIS-HEP [^iris-hep]) and #CNS-1705021, as well as by the Center..."